hbase-shaded/hbase-shaded-client-byo-hadoop/pom.xml

```diff
@@ -52,127 +52,13 @@
         </plugins>
     </build>
     <dependencies>
+        <dependency>
+          <groupId>org.apache.hbase</groupId>
+          <artifactId>hbase-shaded-client-provided-dependencies</artifactId>
+        </dependency>
         <dependency>
             <groupId>org.apache.hbase</groupId>
             <artifactId>hbase-client</artifactId>
         </dependency>
     </dependencies>
-
-    <profiles>
```

Comment:
these were moved verbatim to hbase-shaded-client-provided-dependencies so that they actually get included in the dependency reduced pom (since we use promoteTransitiveDependencies)